services/auto_blog_generator.py

```
@@ -52,67 +52,72 @@ def generate_daily_blog(target_date: Optional[str] = None, upload_to_r2: bool =
         if upload_to_r2:
             try:
                 publisher = R2Publisher()
                 logger.info(f"Uploading blogs to R2...")
-                upload_results = publisher.publish_blogs(Path('blogs'))
+                upload_results = publisher.publish_blogs(Path('data'))
                 
-                # Check if our blog was uploaded
-                blog_key = f'{target_date}/API-v3-{target_date}_digest.json'
+                # Check if our publish package was uploaded
+                blog_key = f'{target_date}/page.publish.json'
                 if blog_key in upload_results and upload_results[blog_key]:
```

Ruff (0.12.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)